### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by DanielSprockett**
I've made some more changes to the readme. (making sure I understand the process)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,38 @@
...
+## Overview
 Snakemake pipeline for basic processing of metagenomic data from the lab.
+
+Module 1: **sn-mg-QC**
+
+Inputs:
+  - Directory of raw fastq files from samples
+  - `File_Paths.txt` document, with 3 columns: `Sample_ID`, `R1`, `R2`
+    - Files will be renamed with `Sample_ID`
+    - `R1` and `R2` should be the full path to the forward and reverse reads, respectively
```
Comment (tanaes) [inline README.md:12]: Wondering if we should add an additional layer with a notion of "sequencing unit", allowing us to include reads from multiple runs.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,38 @@
...
+  - Host genome in fasta format or bowtie2 index thereof
```
Comment (tanaes) [inline README.md:13]: Another stretch goal: could optionally give it an NCBI assembly ID and have it automatically download the appropriate reference. NCBI should have pre-computed indices for most available genomes.

File: README.md [hunk 3]
```diff
@@ -1,3 +1,38 @@
...
+  - `config.yaml` file that defines all of the parameters of the pipeline
+
+Outputs:
+  - Directory of quality filtered fastq files that have been depleted of host reads
+  - Directory of quality filtered fastq files containing only host reads
```
Comment (tanaes) [inline README.md:18]: Maybe BAM alignment instead? Or both?

File: README.md (hunk 1 above)
Comment (DanielSprockett) [reply README.md:12]: Do you mean being able to integrate reads from the same sample that have been sequenced more than once? How would this be handled -- by just concatenating them together?

File: README.md (hunk 2 above)
Comment (DanielSprockett) [reply README.md:13]: I do like this idea. It probably isn't too difficult to implement either. I'll update the README to reflect this, but I guess it is just something that can be specified/described in the `config.yaml` file.

File: README.md (hunk 3 above)
Comment (DanielSprockett) [reply README.md:18]: Ah yes! Definitely output a .BAM file for the host. I'll update this next.

**Pull request merged**